### ISSUE TITLE
 aten::slow_conv3d_forward’ is only available for these backends: [CPU, Autograd, Profiler, Tracer].

### ISSUE THREAD

**Issue by NiharJani2002** [pytorch/pytorch#119122]
### 🐛 Describe the bug

I am using M1 MacBook Pro which has GPU. 
Bcs of Conv3d I have to use CPU, when I shift the code to GPU the above error comes.
I am using latest version of PYTORCH, and specifically the night that is pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu which was installed only 5 days before. 
Hope for help.

### Versions

# packages in environment at /Users/niharjani/Desktop/Miniconda/miniconda3/envs/BRATS:
#
# Name                    Version                   Build  Channel
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
asttokens                 2.4.1              pyhd8ed1ab_0    conda-forge
brotli                    1.1.0                hb547adb_1    conda-forge
brotli-bin                1.1.0                hb547adb_1    conda-forge
brotli-python             1.0.9           py311h313beb8_7  
bzip2                     1.0.8                h620ffc9_4  
ca-certificates           2023.12.12           hca03da5_0  
certifi                   2023.11.17         pyhd8ed1ab_0    conda-forge
cffi                      1.16.0          py311h80987f9_0  
charset-normalizer        2.0.4              pyhd3eb1b0_0  
comm                      0.2.1              pyhd8ed1ab_0    conda-forge
contourpy                 1.2.0           py311hd03642b_0    conda-forge
cryptography              41.0.7          py311hd4332d6_0  
cycler                    0.12.1             pyhd8ed1ab_0    conda-forge
debugpy                   1.8.0           py311ha891d26_1    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
exceptiongroup            1.2.0              pyhd8ed1ab_2    conda-forge
executing                 2.0.1              pyhd8ed1ab_0    conda-forge
ffmpeg                    4.2.2                h04105a8_0  
filelock                  3.13.1          py311hca03da5_0  
fonttools                 4.47.2          py311h05b510d_0    conda-forge
freetype                  2.12.1               hadb7bae_2    conda-forge
fsspec                    2023.12.2                pypi_0    pypi
gettext                   0.21.0               h13f89a0_1  
gmp                       6.2.1                hc377ac9_3  
gmpy2                     2.1.2           py311h40f64dc_0  
gnutls                    3.6.15               h887c41c_0  
hopfield-layers           1.0.2                    pypi_0    pypi
icu                       73.1                 h313beb8_0  
idna                      3.4             py311hca03da5_0  
importlib-metadata        7.0.1              pyha770c72_0    conda-forge
importlib_metadata        7.0.1                hd8ed1ab_0    conda-forge
ipykernel                 6.29.0             pyh3cd1d5f_0    conda-forge
ipython                   8.20.0             pyh707e725_0    conda-forge
itk                       5.3.0                    pypi_0    pypi
itk-core                  5.3.0                    pypi_0    pypi
itk-filtering             5.3.0                    pypi_0    pypi
itk-io                    5.3.0                    pypi_0    pypi
itk-numerics              5.3.0                    pypi_0    pypi
itk-registration          5.3.0                    pypi_0    pypi
itk-segmentation          5.3.0                    pypi_0    pypi
jedi                      0.19.1             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2           py311hca03da5_0  
jupyter_client            8.6.0              pyhd8ed1ab_0    conda-forge
jupyter_core              5.7.1           py311h267d04e_0    conda-forge
kiwisolver                1.4.5           py311he4fd1f5_1    conda-forge
lame                      3.100                h1a28f6b_0  
lcms2                     2.16                 ha0e7c42_0    conda-forge
lerc                      4.0.0                h9a09cb3_0    conda-forge
libblas                   3.9.0           20_osxarm64_openblas    conda-forge
libbrotlicommon           1.1.0                hb547adb_1    conda-forge
libbrotlidec              1.1.0                hb547adb_1    conda-forge
libbrotlienc              1.1.0                hb547adb_1    conda-forge
libcblas                  3.9.0           20_osxarm64_openblas    conda-forge
libcxx                    16.0.6               h4653b0c_0    conda-forge
libdeflate                1.19                 hb547adb_0    conda-forge
libexpat                  2.5.0                hb7217d7_1    conda-forge
libffi                    3.4.4                hca03da5_0  
libgfortran               5.0.0           13_2_0_hd922786_2    conda-forge
libgfortran5              13.2.0               hf226fd6_2    conda-forge
libiconv                  1.16                 h1a28f6b_2  
libidn2                   2.3.4                h80987f9_0  
libjpeg-turbo             3.0.0                hb547adb_1    conda-forge
liblapack                 3.9.0           20_osxarm64_openblas    conda-forge
libopenblas               0.3.25          openmp_h6c19121_0    conda-forge
libopus                   1.3                  h1a28f6b_1  
libpng                    1.6.39               h76d750c_0    conda-forge
libsodium                 1.0.18               h27ca646_1    conda-forge
libsqlite                 3.44.2               h091b4b1_0    conda-forge
libtasn1                  4.19.0               h80987f9_0  
libtiff                   4.6.0                ha8a6c65_2    conda-forge
libunistring              0.9.10               h1a28f6b_0  
libvpx                    1.10.0               hc377ac9_0  
libwebp-base              1.3.2                hb547adb_0    conda-forge
libxcb                    1.15                 hf346824_0    conda-forge
libxml2                   2.10.4               h0dcf63f_1  
libzlib                   1.2.13               h53f4e23_5    conda-forge
llvm-openmp               17.0.6               hcd81f8e_0    conda-forge
markupsafe                2.1.3           py311h80987f9_0  
matplotlib                3.8.2           py311ha1ab1f8_0    conda-forge
matplotlib-base           3.8.2           py311hfdba5f6_0    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mpc                       1.1.0                h8c48613_1  
mpfr                      4.0.2                h695f6f0_1  
mpmath                    1.3.0           py311hca03da5_0  
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
ncurses                   6.4                  h313beb8_0  
nest-asyncio              1.5.9              pyhd8ed1ab_0    conda-forge
nettle                    3.7.3                h84b5d62_1  
networkx                  3.1             py311hca03da5_0  
numpy                     1.26.3          py311h7125741_0    conda-forge
openh264                  1.8.0                h98b2900_0  
openjpeg                  2.5.0                h4c1507b_3    conda-forge
openssl                   3.2.0                h0d3ecfb_1    conda-forge
packaging                 23.2               pyhd8ed1ab_0    conda-forge
pandas                    2.2.0           py311hfbe21a1_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    10.2.0          py311hb9c5795_0    conda-forge
pip                       23.3.1          py311hca03da5_0  
platformdirs              4.1.0              pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.42             pyha770c72_0    conda-forge
psutil                    5.9.8           py311h05b510d_0    conda-forge
pthread-stubs             0.4               h27ca646_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd3eb1b0_0  
pygments                  2.17.2             pyhd8ed1ab_0    conda-forge
pyopenssl                 23.2.0          py311hca03da5_0  
pyparsing                 3.1.1              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1           py311hca03da5_0  
python                    3.11.7          hdf0ec26_1_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.4             pyhd8ed1ab_0    conda-forge
python_abi                3.11                    4_cp311    conda-forge
pytorch                   2.1.2                  py3.11_0    pytorch
pytorch-ranger            0.1.1                    pypi_0    pypi
pytz                      2023.3.post1       pyhd8ed1ab_0    conda-forge
pyyaml                    6.0.1           py311h80987f9_0  
pyzmq                     25.1.2          py311h6727e71_0    conda-forge
readline                  8.2                  h1a28f6b_0  
requests                  2.31.0          py311hca03da5_0  
setuptools                68.2.2          py311hca03da5_0  
simpleitk                 2.3.1                    pypi_0    pypi
six                       1.16.0             pyh6c4a22f_0    conda-forge
sqlite                    3.41.2               h80987f9_0  
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
sympy                     1.12            py311hca03da5_0  
tk                        8.6.13               h5083fa2_1    conda-forge
torch-optimizer           0.3.0                    pypi_0    pypi
torchaudio                2.1.2                 py311_cpu    pytorch
torchvision               0.16.2                py311_cpu    pytorch
tornado                   6.3.3           py311heffc1b2_1    conda-forge
traitlets                 5.14.1             pyhd8ed1ab_0    conda-forge
typing_extensions         4.9.0           py311hca03da5_1  
tzdata                    2023d                h04d1e81_0  
urllib3                   1.26.18         py311hca03da5_0  
wcwidth                   0.2.13             pyhd8ed1ab_0    conda-forge
wheel                     0.41.2          py311hca03da5_0  
x264                      1!152.20180806       h1a28f6b_0  
xorg-libxau               1.0.11               hb547adb_0    conda-forge
xorg-libxdmcp             1.1.3                h27ca646_0    conda-forge
xz                        5.4.5                h80987f9_0  
yaml                      0.2.5                h1a28f6b_0  
zeromq                    4.3.5                h965bd2d_0    conda-forge
zipp                      3.17.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h53f4e23_5    conda-forge
zstd                      1.5.5                h4f39d0f_0    conda-forge

**Comment by malfet** [comment]
Can you provide some sort of minimal reproducer that causes this?

**Comment by NiharJani2002** [comment]
import torch
import torch.nn as nn
tensor = torch.randn(1,1,240,240,155)
conv_layer = Conv3d(input_size = 1, output_size = 10, kernel_size = 3, stride = 3, padding = 1) 
output = conv_layer(tensor)
When executed the code above error generated.

**Comment by malfet** [comment]
Sorry, I can not reproduce the problem using the above script (even after fixing argument names)
By running
```
 python -c "import torch; t=torch.rand(1, 1, 240, 240, 155, device='mps'); conv=torch.nn.Conv3d(in_channels = 1, out_channels = 10, kernel_size = 3, stride = 3, padding = 1).to('mps'); output = conv(t)"
```
I get either get "Conv3D is not supported on MPS" for older PyTorch or valid result using nightly builds
